### PR TITLE
2016-Wellington: Announce, sponsor and CFP changes

### PR DESCRIPTION
**Pull request by DefSol**
- Added the Wellington even to the upcoming NZ one
- Added dates & welcome page
- Added sponsorship artifacts
- Added CFP

### CODE REVIEW

**Comment by phrawzty** [comment]
Ping @bridgetkromhout

**Comment by DefSol** [comment]
Anything in particular?

**Comment by phrawzty** [comment]
> Anything in particular?

She likes to review _new_ event PRs; however, upon closer inspection, this event was already established - you're just updating information. Sorry, I should have paid more attention. I'll review right now. 😄

File: data/events/2016-newzealand.yml [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+cfp_date_start: 2016-06-21  # start accepting talk proposals.
+cfp_date_end: 2016-07-29  # close your call for proposals.
 cfp_date_announce:  # inform proposers of status
-
 # Location
 #
-coordinates: "41.882219, -87.640530" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
-location: "New Zealand" # Defaults to city, but you can make it the venue name.
+coordinates: "-41.279298,174.782064" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "BNZ Harbour Quays, 60 Waterloo Quay, Wellington!" # Defaults to city, but you can make it the venue name.
```
Comment (phrawzty) [inline data/events/2016-newzealand.yml]: Trailing exclamation point?

**Comment by phrawzty** [comment]
👍 with one small nit (above). Please fix and then I'll merge straight away.

**Comment by DefSol** [comment]
Ah, yes that was a hack - when in the md file it renders a space between the data and the exclamation point. I'll remove

**Comment by DefSol** [comment]
Have just updated. Thanks Dan

**Pull request merged**